tests/src/test/java/io/streamnative/pulsar/handlers/kop/KafkaAuthorizationTestBase.java

```diff
@@ -307,7 +307,7 @@ public void testProduceWithTopicLevelPermissions()
         }
     }
 
-    @Test(timeOut = 20000)
+    @Test()
```

Comment:
Can we keep the `timeOut` parameter? Because under normal circumstances, the following logic should not take too long time.